cluster/gce/configure-vm.sh

```diff
@@ -453,7 +456,11 @@ if [[ -z "${is_push}" ]]; then
   set-kube-env
   [[ "${KUBERNETES_MASTER}" == "true" ]] && mount-master-pd
   create-salt-pillar
-  create-salt-auth
+  [[ "${KUBERNETES_MASTER}" == "true" ]]; then
```

Comment:
Actually, I'm surprised this doesn't just cause nodes to fail to boot with errexit.